docs/architecture.md

```diff
@@ -4,38 +4,54 @@ title: Architecture
 hide_title: false
 ---
 
+## Architecture
+
 Below, find a diagram with the components that comprise an instance of Cumulus.
 
 ![Architecture diagram of a Cumulus deployment](assets/cumulus-arch-diagram.png)
 
-While the internals are complex, the boxes represent components which are easy to understand:
+This diagram details all of the major architectural  components of a Cumulus deployment.
+
+While the diagram can feel complex, it can easily be digested in several major components:
+
+### Data Distribution
+
+End Users can access data via Cumulus's `distribution` submodule, which includes ASF's [thin egress application](https://github.com/asfadmin/thin-egress-app), this provides authenticated data egress, temporary S3 links and other statistics features.
+
+### Data ingest
+
+#### Workflows
+
+The core of the ingest and processing capabilities in Cumulus is built into the deployed AWS [Step Function](https://aws.amazon.com/step-functions/) workflows.    Cumulus rules trigger workflows via either Cloud Watch rules, Kinesis streams, SNS topic, or SQS queue.   The workflows then run with a configured [Cumulus message](./workflows/cumulus-task-message-flow), utilizing built-in processes to report status of granules, PDRs, executions, etc to the [Data Persistence](#data-persistence) components.
+
+Workflows can optionally report granule status to [CMR](<https://earthdata.nasa.gov/eosdis/science-system-description/eosdis-components/cmr>), and if configured and using the [Cumulus message adapter (CMA)](<https://github.com/nasa/cumulus-message-adapter>), workflow steps can report metrics information to a shared SNS topic, for export to an external ELK stack to make use of.
```

Comment:
```suggestion
Workflows can optionally report granule metadata to [CMR](<https://earthdata.nasa.gov/eosdis/science-system-description/eosdis-components/cmr>), and if configured to use the [Cumulus message adapter (CMA)](<https://github.com/nasa/cumulus-message-adapter>), workflow steps can report metrics information to a shared SNS topic, which could be subscribed to for near real time granule, execution, and PDR status. This could be used for metrics reporting using an external ELK stack, for example.
```

Reply:
So you have to use the CMA to have it post to the SNS topic?

Reply:
@laurenfrederick my understanding is the cloudwatch logs won't have the metadata needed for stats reporting if the CMA is not involved.   @flamingbear ?

Reply:
lauren did those tickets.  But my understanding is cloudwatch logs will need the CMA enabled in order to get the enhanced logs from the lambda tasks.  But I don't think that's the case for the SNS "logs" (messages send to sns), I think those are standalone.  I haven't looked at them yet though.

Reply:
Hmm, I see, I think I have a misunderstanding of what that portion of the diagram implies.     Sounds like we should pull out the portion about the CMA?

Reply:
Yeah, the CMA is not required for that as far as i know. It's based on the output of the workflow.